libraries/c_sdk/standard/mqtt/test/unit/iot_tests_mqtt_api.c

```diff
@@ -828,7 +826,7 @@ TEST( MQTT_Unit_API, OperationWaitTimeout )
         TEST_ASSERT_NOT_NULL( _pMqttConnection );
 
         /* Set the MQTT Context for the new MQTT Connection*/
-        TEST_ASSERT_EQUAL( IOT_MQTT_SUCCESS, _setContext( _pMqttConnection, _sendDelay ) );
+        TEST_ASSERT_EQUAL( IOT_MQTT_SUCCESS, _setContext( _pMqttConnection, ( TransportSend_t ) _sendDelay ) );
```

Comment:
Instead of a explicit casting, can their be 2 different `_sendDelay` functions, one that adheres to network abstraction's send signature, and the other that adheres to the new `TransportSend_t`'s?

Reply:
I have thought that as an option too. I think that would be a better option. Will add that fix.